src/pcapi/routes/pro/venues.py

```diff
@@ -82,6 +84,10 @@ def edit_venue(venue_id: str, body: EditVenueBodyModel) -> GetVenueResponseModel
     check_user_has_access_to_offerer(current_user, venue.managingOffererId)
     venue = offerers_api.update_venue(venue, **body.dict(exclude_unset=True))
```

Comment:
```suggestion
    venue = offerers_api.update_venue(venue, **body.dict(exclude={'isWithdrawalAppliedOnAllOffers'}, exclude_unset=True))
```
Ainsi `update_venue` n'a pas à se préoccuper de ce champs de formulaire qui ne concerne que cette route.